projects/kit/components/toggle/toggle.template.html

```diff
@@ -7,33 +7,52 @@
     [invalid]="computedInvalid"
 >
     <div class="toggle">
-        <tui-loader
-            *ngIf="showLoader"
-            class="loader"
-            [size]="loaderSize"
-            [inheritColor]="true"
-            [showLoader]="checked"
-        ></tui-loader>
-        <tui-svg
-            *ngIf="showIcons && !showLoader"
-            class="icon"
-            automation-id="tui-toggle__check-icon"
-            [src]="iconOn"
-        ></tui-svg>
+        <span
+            polymorpheus-outlet
+            class="icon-wrapper"
+            [content]="iconOn"
+            [context]="context"
```

Comment:
I recently discovered that Angular optimizes inline objects, so we can just write `{ $implicit: size }` here and drop getter and pure private method.

Reply:
Wow! That's cool. Changed